[Bug]: pnpm installation instructions fail

## Summary

What went wrong?

## Steps to reproduce

```
npm install -g openclaw@latest
# or: pnpm add -g openclaw@latest

openclaw onboard --install-daemon

The wizard installs the Gateway daemon (launchd/systemd user service) so it stays running.
---
## Expected behavior

Instructions to work.

## Actual behavior

```
Davids-iMac.local-root# openclaw onboard --install-daemon
 ERR_PNPM_NO_IMPORTER_MANIFEST_FOUND  No package.json (or package.yaml, or package.json5) was found in "/Users/bigdavid".
```

## Environment

- Clawdbot version: No way to know, as openclaw fails to run
- OS:o sx-12.7
- Install method (pnpm/npx/docker/etc):

```
sudo bash
Davids-iMac.local-root# curl -fsSL https://get.pnpm.io/install.sh | sh -
==> Downloading pnpm binaries 10.28.2
 WARN  using --force I sure hope you know what you are doing
Copying pnpm CLI from /private/var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.VwZVAcrc/pnpm to /Users/bigdavid/Library/pnpm/.tools/pnpm-exe/10.28.2/pnpm
No changes to the environment were made. Everything is already up to date.

Davids-iMac.local-root# pnpm setup
No changes to the environment were made. Everything is already up to date.

Davids-iMac.local-root# pnpm add -g openclaw@latest
Downloading openclaw@2026.1.30: 12.80 MB/12.80 MB, done
Downloading pdfjs-dist@5.4.530: 9.88 MB/9.88 MB, done
Downloading @img/sharp-libvips-darwin-x64@1.2.4: 8.27 MB/8.27 MB, done
Packages are cloned from the content-addressable store to the virtual store.
  Content-addressable store is at: /Users/bigdavid/Library/pnpm/store/v10
  Virtual store is at:             .pnpm
Downloading @napi-rs/canvas-darwin-x64@0.1.89: 13.30 MB/13.30 MB, done
Downloading node-llama-cpp@3.15.1: 28.13 MB/28.13 MB, done
Downloading @node-llama-cpp/mac-x64@3.15.1: 7.84 MB/7.84 MB, done
 WARN  5 deprecated subdependencies found: are-we-there-yet@3.0.1, gauge@4.0.4, node-domexception@1.0.0, npmlog@6.0.2, tar@6.2.1
Packages: +623
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 691, reused 0, downloaded 624, added 623, done

/Users/bigdavid/Library/pnpm/global/5:
+ openclaw 2026.1.30

╭ Warning ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│                                                                                                                                                             │
│   Ignored build scripts: @whiskeysockets/baileys@7.0.0-rc.9, node-llama-cpp@3.15.1, openclaw@2026.1.30, protobufjs@6.8.8, protobufjs@7.5.4, sharp@0.34.5.   │
│   Run "pnpm approve-builds -g" to pick which dependencies should be allowed to run scripts.                                                                 │
│                                                                                                                                                             │
╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
Done in 12.9s using pnpm v10.28.2

Davids-iMac.local-root# openclaw onboard --install-daemon
 ERR_PNPM_NO_IMPORTER_MANIFEST_FOUND  No package.json (or package.yaml, or package.json5) was found in "/Users/bigdavid".
```
## Logs or screenshots

Paste relevant logs or add screenshots (redact secrets).

--- a/docs/install/index.md
+++ b/docs/install/index.md
@@ -73,11 +73,14 @@ SHARP_IGNORE_GLOBAL_LIBVIPS=1 npm install -g openclaw@latest
 
 If you see `sharp: Please add node-gyp to your dependencies`, either install build tooling (macOS: Xcode CLT + `npm install -g node-gyp`) or use the `SHARP_IGNORE_GLOBAL_LIBVIPS=1` workaround above to skip the native build.
 
-Or:
+Or with pnpm:
 
 ```bash
 pnpm add -g openclaw@latest
+pnpm approve-builds -g   # select openclaw, node-llama-cpp, sharp, etc.
 ```
+
+pnpm requires explicit approval for packages with build scripts. After `pnpm add -g`, run `pnpm approve-builds -g` and approve the listed packages (openclaw, node-llama-cpp, sharp, protobufjs, etc.), then pnpm will run their postinstall scripts.
 
 Then:
 

--- a/docs/install/index.md
+++ b/docs/install/index.md
@@ -77,10 +77,11 @@ Or with pnpm:
 
 ```bash
 pnpm add -g openclaw@latest
-pnpm approve-builds -g   # select openclaw, node-llama-cpp, sharp, etc.
+pnpm approve-builds -g                # approve openclaw, node-llama-cpp, sharp, etc.
+pnpm add -g openclaw@latest           # re-run to execute postinstall scripts
 ```
 
-pnpm requires explicit approval for packages with build scripts. After `pnpm add -g`, run `pnpm approve-builds -g` and approve the listed packages (openclaw, node-llama-cpp, sharp, protobufjs, etc.), then pnpm will run their postinstall scripts.
+pnpm requires explicit approval for packages with build scripts. After the first install shows the "Ignored build scripts" warning, run `pnpm approve-builds -g` and select the listed packages, then re-run the install so postinstall scripts execute.
 
 Then:
 

